test/test_jit_cuda_fuser.py

```diff
@@ -2186,13 +2186,14 @@ def test_gelu(self):
         x = torch.randn([1024, 1024], dtype=dtype, device=device, requires_grad=True)
         grads = torch.randn([1024, 1024], dtype=dtype, device=device, requires_grad=False)
 
-        def t(x: torch.Tensor):
-            o = torch.nn.functional.gelu(x)
+        def t(x: torch.Tensor, mode : str):
+            o = torch.nn.functional.gelu(x, approximate=mode)
             o = o * 2.0
             return o
 
         t_jit = torch.jit.script(t)
-        self._run_training_helper(t_jit, t, grads, x)
+        self._run_training_helper(t_jit, t, grads, x, 'none')
```

Comment:
Does this actually pass the test?!

We disabled fusion guard in CI tests, that means when you change `mode`, it's running the same graph and ignores the `tanh` passed in the second time.

You might want to turn on guard mode again to run both mode in a single test like this one: https://github.com/pytorch/pytorch/blob/c2cd1d3219486dbf07662bff2a3874664c8f6978/test/test_jit_cuda_fuser.py#L974